espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/model/ViewElement.java

```diff
@@ -0,0 +1,104 @@
+package io.appium.espressoserver.lib.model;
+
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+
+public class ViewElement {
+
+    private final int id;
+    private final boolean clickable;
+    private final boolean longClickable;
+    private final boolean focused;
+    private final String className;
+    private int index;
+    private String contentDescription = "";
+    private String bounds;
+
+    public ViewElement(View view) {
+        // Get content description
+        if (view.getContentDescription() != null)
+            this.contentDescription = view.getContentDescription().toString();
+
+        // Get bounds
+        int[] l = new int[2];
+        view.getLocationOnScreen(l);
+        this.bounds = String.format("[%d, %d],[%d, %d]", l[0], l[1], l[0] + view.getWidth(), l[1] + view.getHeight());
+
+        // Get ID
+        id = view.getId(); // TODO: Not sure if we need this. Will leave for now
+
+        // Get className
+        className = view.getClass().getName();
+
+        // Get the index
+        ViewParent parent = view.getParent();
+        try {
+            for (int index = 0; index < ((ViewGroup) parent).getChildCount(); ++index) {
+                View childView = ((ViewGroup) parent).getChildAt(index);
+                if (childView.equals(view)) {
+                    this.index = index;
+                }
+            }
+        } catch (ClassCastException e) {
+            // If it couldn't be cast to a ViewGroup, the parent has no children
+        }
+
+
+
+        // Get booleans
+        clickable = view.isClickable();
+        longClickable = view.isLongClickable();
+        focused = view.isAccessibilityFocused();
+
+        // TODO: Attributes that need to be added with examples
+        // resource-id android:id/decor_content_parent
+        // index 0
+        // text "Hello World"
+        // package	io.appium.android.apis
+        // content-desc
+        // checkable	false
+        // checked	false
+        // enabled	true
+        // focusable	false
+        // scrollable	false
+        // password	false
+        // selected	false
+        // bounds	[0,0][1440,2560]
+        // resource-id	android:id/decor_content_parent
+        // instance	0
+
+    }
+
+    public String getContentDescription() {
```

Comment:
I'd not return strings here, but rather "original" types. We can convert them inside xml writer if we need it. In the worst case one my just box simple types like boolean or int into Boolean/Integer and then call toString().toLowerCase(), where needed.

There will be another API calls like isClickable, which should return boolean value, so we will have to perform "reverse" type conversation again there